rest/route/project.go

```diff
@@ -607,8 +607,12 @@ func (h *projectIDPutHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "unmarshalling JSON request body into project ref"))
 	}
 
-	dbProjectRef := apiProjectRef.ToService()
-	dbProjectRef.Identifier = h.projectName
+	dbProjectRef := dbModel.ProjectRef{
+		Identifier: h.projectName,
+		Id:         utility.FromStringPtr(apiProjectRef.Id),
```

Comment:
Can we add validation for the owner? (i.e. that it's within the allowed organizations)

Reply:
Done, and I added it to the graphql resolver as well.